apps/jobs/views.py

```diff
@@ -322,6 +322,40 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
         )
 
 
+@api_view(["DELETE"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def challenge_submission_delete(request, challenge_pk, challenge_phase_pk, submission_pk):
+    """
+    Delete submission by the challenge Host
```

Comment:
@Kajol-Kumari please use proper docstrings and add a description for all the params to the api. Please see other APIs for reference